src/utils/network/utils.js

```diff
@@ -244,6 +244,30 @@ export function buildURL({ AMaaSClass, AMId, resourceId, stage, apiVersion }) {
         apiVersion
       })}/transaction/aggregate_pnls`
       break
+    case 'eod':
+      baseURL = `${getEndpoint({
+        stage,
+        apiVersion
+      })}/marketdata/eod-prices`
+        break
+    case 'curve':
+      baseURL = `${getEndpoint({
+        stage,
+        apiVersion
+      })}/marketdata/curves`
+      break
+    case 'fxRate':
+      baseURL = `${getEndpoint({
+        stage,
+        apiVersion
+      })}/marketdata/fx-rates`
+      break
+    case 'forwardRate':
+      baseURL = `${getEndpoint({
+        stage,
+        apiVersion,
+      })}/marketdata/forward-rates/${resourceId}`
```

Comment:
There should not be a `resourceId` here. Look at lines 274 - 279 to see how `AMId` and `resourceId` are appended to the `baseURL`.